phpBB/phpbb/path_helper.php

```diff
@@ -100,11 +100,17 @@ public function get_php_ext()
 	*/
 	public function update_web_root_path($path)
 	{
+		$web_root_path = $this->get_web_root_path();
+
+		if (strpos($path, $web_root_path) === 0)
```

Comment:
Maybe, since this is a bit of a cheat, add a comment explaining why this is done?